packages/bierzo-wallet/src/routes/registerName/index.tsx

```diff
@@ -117,16 +117,17 @@ function getChainAddressPairsFromValues(
 
     const type = key.substr(fieldValueIdxLength + 1);
     switch (type) {
-      case addressValueField:
+      case addressValueField: {
         pair = { ...pair, address: values[key] as Address };
         break;
-      case blockchainValueField:
-        // eslint-disable-next-line no-case-declarations
+      }
```

Comment:
The brackets are only required when you declare variables in the block (like `const chain = ...` below).

In `case addressValueField:` the brackets are not needed but it is also not wrong. Maybe linter complains, let's see ...